src/ui/pages/Menu/Menu.tsx

```diff
@@ -129,6 +137,18 @@ const Menu = () => {
             label={`${i18n.t("menu.tab.items.credentials")}`}
           />
         </IonRow>
+        <IonRow className="menu-input-row">
+          <MenuItem
+            index={4}
+            icon={lockClosedOutline}
+            label={`${i18n.t("menu.tab.items.lock")}`}
+          />
+          <MenuItem
+            index={5}
+            icon=""
+            label=""
+          />
+        </IonRow>
```

Comment:
2 questions here:
- If we are adding a lock button to the menu - has it gone through Robert to update designs and agree? Don't remember talking about locking the app.
- We did this trick to add an empty menu item for the tunnel branch, but now this on develop. I don't think we should have these empty menu items.

Reply:
Cool, if we wait until then we can either re-open the story or create a new task to address these items. As long as its on the board so we remember.